x-pack/legacy/plugins/siem/public/components/matrix_histogram/index.test.tsx

```diff
@@ -52,36 +68,56 @@ describe('Matrix Histogram Component', () => {
     subtitle: 'mockSubtitle',
     totalCount: -1,
     title: 'mockTitle',
-    updateDateRange: jest.fn(),
+    dispatchSetAbsoluteRangeDatePicker: jest.fn(),
   };
-  describe('rendering', () => {
-    test('it renders EuiLoadingContent on initialLoad', () => {
-      const wrapper = shallow(<MatrixHistogram {...mockMatrixOverTimeHistogramProps} />);
 
-      expect(wrapper.find(`[data-test-subj="initialLoadingPanelMatrixOverTime"]`)).toBeTruthy();
+  beforeAll(() => {
+    (useQuery as jest.Mock).mockReturnValue({
+      data: null,
+      loading: false,
+      inspect: false,
+      totalCount: null,
     });
-
-    test('it renders Loader while fetching data if visited before', () => {
-      const mockProps = {
-        ...mockMatrixOverTimeHistogramProps,
-        data: [{ x: new Date('2019-09-16T02:20:00.000Z').valueOf(), y: 3787, g: 'config_change' }],
-        totalCount: 10,
-        loading: true,
-      };
-      const wrapper = shallow(<MatrixHistogram {...mockProps} />);
-      expect(wrapper.find('.loader')).toBeTruthy();
+    wrapper = mount(<MatrixHistogram {...mockMatrixOverTimeHistogramProps} />);
+  });
+  describe('on initial load', () => {
+    test('it renders MatrixLoader', () => {
+      expect(wrapper.html()).toMatchSnapshot();
+      expect(wrapper.find('MatrixLoader')).toHaveLength(1);
```

Comment:
Assertions in the form of:

```
expect(wrapper.find('thing')).toHaveLength(1);
```

like this one and the others in this file / PR, are often another way of saying "`thing` exists".  When applicable, consider using `exists`-style assertions, e.g.: 

```
expect(wrapper.find('MatrixLoader').exists()).toBe(true);
```